docs/core/v3.0.rst

```diff
@@ -0,0 +1,1763 @@
+.. This file is in restructured text format: http://docutils.sourceforge.net/rst.html
+.. _zarr-core-specification-v3.0:
+
+======================================
+ Zarr core specification (version 3.0)
+======================================
+-------------------------------
+ Editor's draft 25 May 2022
+-------------------------------
+
+Specification URI:
+    https://purl.org/zarr/spec/core/3.0
+    
+Editors:
+    * Alistair Miles (`@alimanfoo <https://github.com/alimanfoo>`_), Wellcome Sanger Institute
+    * Jonathan Striebel (`@jstriebel <https://github.com/jstriebel>`_), Scalable Minds
+
+Issue tracking:
+    `GitHub issues <https://github.com/zarr-developers/zarr-specs/labels/core-spec-v3.0>`_
+
+Suggest an edit for this spec:
+    `GitHub editor <https://github.com/zarr-developers/zarr-specs/blob/core-protocol-v3.0-dev/docs/core/v3.0.rst>`_
+
+Copyright 2019-Present `Zarr core development team
+<https://github.com/orgs/zarr-developers/teams/core-devs>`_. This work
+is licensed under a `Creative Commons Attribution 3.0 Unported License
+<https://creativecommons.org/licenses/by/3.0/>`_.
+
+----
+
+
+Abstract
+========
+
+This specification defines the Zarr format for N-dimensional typed arrays.
+
+
+Status of this document
+=======================
+
+This document is a **Work in Progress**. It may be updated, replaced
+or obsoleted by other documents at any time.
+
+Comments, questions or contributions to this document are very
+welcome. Comments and questions should be raised via `GitHub issues
+<https://github.com/zarr-developers/zarr-specs/labels/core-spec-v3.0>`_.
+
+This document was produced by the `Zarr core development team
+<https://github.com/orgs/zarr-developers/teams/core-devs>`_.
```

Comment:
```suggestion
```

Not sure the best way to capture contributors to the spec, but suggest removing this for now.

Reply:
Updated accordingly in #168. Marking as resolved here.